src/containers/pages/IRS/Map/index.tsx

```diff
@@ -255,6 +256,20 @@ const IRSReportingMap = (props: IRSReportingMapProps & RouteComponentProps<Route
 
   const gisidaWrapperProps = getGisidaWrapperProps(jurisdiction, structures, indicatorStops);
 
+  /**
+   * Create list elements from dictionary
+   * @param {Dictionary} dict
+   */
+  const processListDisplay = (dict: Dictionary | string) => {
+    const createList = (obj: Dictionary) =>
+      Object.keys(obj).map(key => (
```

Comment:
Can we use [Object.entries](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries)?  Is it better?

Reply:
Yah! it is better.